Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -482,10 +476,10 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
 }
 
 - (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
-             isAdaptiveAdViewEnabled:(BOOL)isAdaptiveAdViewEnabled
                           parameters:(id<MAAdapterParameters>)parameters
+             isVungleAdaptiveEnabled:(BOOL)isVungleAdaptiveEnabled
 {
-    if ( isAdaptiveAdViewEnabled && [self isAdaptiveAdViewFormat: adFormat forParameters: parameters] )
+    if ( isVungleAdaptiveEnabled && [self isAdaptiveAdViewFormat: adFormat forParameters: parameters] )
```

Comment:
need to also check if Vungle SDK supports adaptive for this placement ID. hence this new Bool.